tutor4points/core/templates/points.html

```diff
@@ -9,26 +9,27 @@
 {% block heading %}
 Points
 <font color="red">
-{% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
+    {% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
 </font>
 {% endblock %}
 
 <!-- {% load crispy_forms_tags %} -->
 {% block content %}
 <div class="container-fluid main-container">
+    <div class="alert alert-success hidden" id="success-box" role="alert">
+        {{success_message}}
```

Comment:
Success alert is always rendered on points page even when there's no action taken.